### DISCUSSION TITLE
[Question] Count entries of multi-select combo box

### DISCUSSION THREAD

**Discussion by line-ankaa** [YetiForceCompany/YetiForceCRM#13974] [How to]
Hello there,

I am really sorry to bother you with my question, because it might seem a bit stupid. I am very new to yetiforce, do not have any programming experience and I am working for a small non-profit organisation (vocational training centre).

I have a module with all data of all my students and I created a multi-select combo box with a list of all the classes that my students might have taken in 2018-2019, which will be updated by importing different Exel sheets. Now, I added a new field (integer) called number of classes in 2018-2019. I would like through a workflow, that this gets done automatically. So basically, if there are three different classes selected in my multi-select combo box, that the field classes_2018 gives me the number 3.
I do not know if I am clear, but is there somebody out there that can help me? It sounds like a very basic/stupid thing, but I am not able to find the right answer.

Looking forward to your help :)

**Comment by vovpff** [comment]
I don't know about workflow but this can be done by simple selfwtitten handler.

**Reply by line-ankaa** [reply]
Thank you for your reply. Can you please explain what you mean by "simple self-written handler"? Thank you so much

**Reply by vovpff** [reply]
@line-ankaa pls email me with more detailed explanation of your task and screenshots of your system and fields.

**Comment by vovpff** [comment]
@line-ankaa pls check your email
